### ISSUE TITLE
Make fails on MacOS Big Sur

### ISSUE THREAD

**Issue by gitscosh** [st3w/neo#16]
I have the latest version of command line tools available to my OS verison.
```
 ~/Scr/neo-0.6  make                                                                                                                     10.3s  Tue Dec 21 02:03:51 2021
Making all in doc
make[1]: Nothing to be done for `all'.
Making all in src
g++ -std=c++11 -DPACKAGE_NAME=\"neo\" -DPACKAGE_TARNAME=\"neo\" -DPACKAGE_VERSION=\"0.6\" -DPACKAGE_STRING=\"neo\ 0.6\" -DPACKAGE_BUGREPORT=\"\" -DPACKAGE_URL=\"\" -DPACKAGE=\"neo\" -DVERSION=\"0.6\" -DHAVE_LIBNCURSESW=1 -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_GETOPT_H=1 -DHAVE_LOCALE_H=1 -DHAVE_NCURSES_H=1 -DHAVE_CXX11=1 -I.   -I/usr/local/opt/ncurses/include -DNCURSES_WIDECHAR -std=c++11 -g -O2 -MT cloud.o -MD -MP -MF .deps/cloud.Tpo -c -o cloud.o cloud.cpp
In file included from cloud.cpp:20:
./cloud.h:167:18: error: chosen constructor is explicit in copy-initialization
    mt19937 mt = {};
                 ^~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/random:2110:14: note: explicit constructor declared here
    explicit mersenne_twister_engine(result_type __sd = default_seed)
             ^
In file included from cloud.cpp:20:
./cloud.h:168:52: error: chosen constructor is explicit in copy-initialization
    uniform_int_distribution<int> _randColorPair = {};
                                                   ^~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/algorithm:3008:14: note: explicit constructor declared here
    explicit uniform_int_distribution(result_type __a = 0,
             ^
In file included from cloud.cpp:20:
./cloud.h:169:52: error: chosen constructor is explicit in copy-initialization
    uniform_real_distribution<float> _randChance = {};
                                                   ^~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/random:3730:14: note: explicit constructor declared here
    explicit uniform_real_distribution(result_type __a = 0, result_type __b = 1)
             ^
In file included from cloud.cpp:20:
./cloud.h:170:52: error: chosen constructor is explicit in copy-initialization
    uniform_int_distribution<uint16_t> _randLine = {};
                                                   ^~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/algorithm:3008:14: note: explicit constructor declared here
    explicit uniform_int_distribution(result_type __a = 0,
             ^
In file included from cloud.cpp:20:
./cloud.h:171:53: error: chosen constructor is explicit in copy-initialization
    uniform_int_distribution<uint16_t> _randCpIdx = {};
                                                    ^~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/algorithm:3008:14: note: explicit constructor declared here
    explicit uniform_int_distribution(result_type __a = 0,
             ^
In file included from cloud.cpp:20:
./cloud.h:172:51: error: chosen constructor is explicit in copy-initialization
    uniform_int_distribution<uint16_t> _randLen = {};
                                                  ^~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/algorithm:3008:14: note: explicit constructor declared here
    explicit uniform_int_distribution(result_type __a = 0,
             ^
In file included from cloud.cpp:20:
./cloud.h:173:51: error: chosen constructor is explicit in copy-initialization
    uniform_int_distribution<uint16_t> _randCol = {};
                                                  ^~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/algorithm:3008:14: note: explicit constructor declared here
    explicit uniform_int_distribution(result_type __a = 0,
             ^
In file included from cloud.cpp:20:
./cloud.h:174:56: error: chosen constructor is explicit in copy-initialization
    uniform_int_distribution<uint16_t> _randGlitchMs = {};
                                                       ^~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/algorithm:3008:14: note: explicit constructor declared here
    explicit uniform_int_distribution(result_type __a = 0,
             ^
In file included from cloud.cpp:20:
./cloud.h:175:56: error: chosen constructor is explicit in copy-initialization
    uniform_int_distribution<uint16_t> _randLingerMs = {};
                                                       ^~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/algorithm:3008:14: note: explicit constructor declared here
    explicit uniform_int_distribution(result_type __a = 0,
             ^
In file included from cloud.cpp:20:
./cloud.h:176:53: error: chosen constructor is explicit in copy-initialization
    uniform_int_distribution<size_t> _randCharIdx = {};
                                                    ^~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/algorithm:3008:14: note: explicit constructor declared here
    explicit uniform_int_distribution(result_type __a = 0,
             ^
In file included from cloud.cpp:20:
./cloud.h:177:51: error: chosen constructor is explicit in copy-initialization
    uniform_real_distribution<float> _randSpeed = {};
                                                  ^~
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/random:3730:14: note: explicit constructor declared here
    explicit uniform_real_distribution(result_type __a = 0, result_type __b = 1)
             ^
11 errors generated.
make[1]: *** [cloud.o] Error 1
make: *** [all-recursive] Error 1
 2  ~/Scr/neo-0.6 
```

**Comment by st3w** [comment]
This was first seen on FreeBSD in issue #8. It was fixed in commit 4dd64a0bc31bdd1cc4f5401a7a2d37076329923e. I haven't made another neo release just yet because I'm waiting for confirmation of the fixes for issue #9 also.

**Comment by gitscosh** [comment]
Understood. I'll pull down that commit then. Thanks!
